### DISCUSSION TITLE
Add option to change default port

### DISCUSSION THREAD

**Discussion by homehtha** [louislam/dockge#129] [Ask for Help] [answered]
Please add option to change default port (5001).

**Comment by trythatagain** [comment]
Are you on about the host side (left port number),    because you can change the left hand port to whatever you want i.e 5011. Or is there some other container that uses 5001 as their default port?

 ports:
      # Host Port:Container Port
      - 5001:5001

**Reply by homehtha** [reply]
You are talking about mapped port (if you are using this container on the same address as host).
I am talking about the option to change default port (in your example right side), this is done by adding environment variable (usually PORT). You can not use "ports" option in segmented network (VLANs, each container has own address).

**Reply by homehtha** [reply]
@louislam 
I unmarked as answer for reasons stated above.

**Comment by louislam** [comment]
Can be changed by this environment variable: `DOCKGE_PORT=5001`

I don't have time to write the wiki yet, but you can find more environment variables here.
https://github.com/louislam/dockge/blob/0279d431e06afe57ae754c108f5e266468482c08/backend/dockge-server.ts#L129

**Reply by homehtha** [reply]
Sadly it doesn't work (I have tried it with latest 1.1.1 version).
Even if I add different port, it's ignored (still 5001).

EDIT: Also tried version 1.0.4 - same result :(

**Reply by louislam** [reply, marked as answer]
Just realized it is hard-coded. Fixed, but need to wait for the next release.
https://github.com/louislam/dockge/commit/81cacbddddce1bae4da0142cdad18d2c7479c43b
